dotnet/src/SemanticKernel.Core/Extensions/KernelSemanticFunctionExtensions.cs

```diff
@@ -118,7 +118,7 @@ public static class KernelSemanticFunctionExtensions
         return kernel.RunAsync(skfunction);
     }
 
-    [Obsolete("Methods and classes which isnclud Skill in the name have been renamed to use Plugin. Use Kernel.ImportSemanticFunctionsFromDirectory instead. This will be removed in a future release.")]
+    [Obsolete("Methods and classes which isnclud Skill in the name have been renamed to use Plugin. Use Kernel.ImportSemanticPluginFromDirectory instead. This will be removed in a future release.")]
```

Comment:
@lemillermicrosoft I was chatting with Matthew last night and he has requested we use the ImportSemanticFunctionsFromDirectory naming. So this PR will be superseded by this PR https://github.com/microsoft/semantic-kernel/pull/2937.